lib/app/routes/exercises.rb

```diff
@@ -133,6 +141,51 @@ class Exercises < Core
         Hack::UpdatesUserExercise.new(submission.user_id, submission.track_id, submission.slug).update
         redirect "/"
       end
+
+      get '/submissions/blob/content' do
+        content_type :json
+        submission = Submission.find_by_key(params[:key])
+        blob = Octokit.blob("#{submission.user.username}/#{submission.slug}", params[:sha])
+        s = SourceClassifier.new(File.join(File.dirname(__FILE__), '../../../bin/', 'trainer.bin'))
```

Comment:
Line is too long. [99/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.